.circleci/config.yml

```diff
@@ -508,101 +518,54 @@ parameters:
     type: boolean
     default: false
 
-  target-env:
-    type: string
-    default: ''
-  owasp:
-    type: boolean
-    default: false
-
-  qasp:
+  run_owasp_scan:
     type: boolean
     default: false
 
-  pa11y:
-    type: boolean
-    default: false
-
-  frontend:
-    type: boolean
-    default: false
-
-  backend:
-    type: boolean
-    default: false
+  target_env:
+    type: string
+    default: ''
 
 workflows:
-  nightly:
-    jobs:
-      - nightly-owasp-scan
-    triggers:
-      - schedule:
-          cron: "0 0 * * *"
-          filters:
-            branches:
-              only:
-               - raft-tdp-main
-
   build-and-test:
-
-    unless: << pipeline.parameters.run_dev_deployment >> 
-  # Remove this check in a follow up PR to the work for 1013
+    unless:
+      or:
+        - << pipeline.parameters.run_dev_deployment >>
+        - << pipeline.parameters.run_owasp_scan >>
```

Comment:
Prevent running `build-and-test` again when the API call comes in for the OWASP scan.